### ISSUE TITLE
What would it take to support u24 or u32 glyph ids?

### ISSUE THREAD

**Issue by rsheeter** [googlefonts/fontations#118]
Just want to be sure we aren't painting ourselves into a corner.

**Comment by rsheeter** [comment]
For reading being able to just always have u32 seems fine. Presumably the compiler needs direct control over width. If we u32 on read would that impede the compiler?

**Comment by cmyr** [comment]
nope, codegen would know the underlying types and so would generate the correct representation for compilation.
